tests/audio/test_pit.py

```diff
@@ -0,0 +1,167 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+from functools import partial
+from typing import Callable
+
+import numpy as np
+import pytest
+import torch
+from scipy.optimize import linear_sum_assignment
+from torch import Tensor
+
+from tests.helpers import seed_all
+from tests.helpers.testers import BATCH_SIZE, NUM_BATCHES, MetricTester
+from torchmetrics.audio import PIT
+from torchmetrics.functional import pit, si_sdr, snr
+from torchmetrics.utilities.imports import _TORCH_GREATER_EQUAL_1_6
+
+seed_all(42)
+
+Time = 10
+
+Input = namedtuple('Input', ["preds", "target"])
+
+inputs1 = Input(
+    preds=torch.rand(NUM_BATCHES, BATCH_SIZE, 3, Time),
+    target=torch.rand(NUM_BATCHES, BATCH_SIZE, 3, Time),
+)
+inputs2 = Input(
+    preds=torch.rand(NUM_BATCHES, BATCH_SIZE, 2, Time),
+    target=torch.rand(NUM_BATCHES, BATCH_SIZE, 2, Time),
+)
+
+
+def scipy_version(preds: Tensor, target: Tensor, metric_func: Callable, eval_func: str):
```

Comment:
docstring what does the func do? what is the return type?
by the name, I expect return scipy version but it does not seem so...

Reply:
see https://github.com/PyTorchLightning/metrics/pull/384/commits/29744ab59371fdef6fd1e3b7b803b2a72b6078a8